src/ocamlorg_package/lib/ocamlorg_package.ml

```diff
@@ -12,11 +12,14 @@ let version t = t.version
 let info t = t.info
 let create ~name ~version info = { name; version; info }
 
+module DocumentationStatusMap = Map.Make (String)
+
```

Comment:
```suggestion
```

seems obsolete now that we're using `Documentation_status.t option Version.Map.t Name.Map.t`

Reply:
yeah, need to remove that one.